openpype/hosts/traypublisher/plugins/publish/collect_csv_ingest_instance_data.py

```diff
@@ -0,0 +1,36 @@
+from pprint import pformat
+import pyblish.api
+from openpype.pipeline import publish
+
+
+class CollectCSVIngestInstancesData(
+    pyblish.api.InstancePlugin,
+    publish.OpenPypePyblishPluginMixin,
+    publish.ColormanagedPyblishPluginMixin
+):
+    """Collect CSV Ingest data from instance.
+    """
+
+    label = "Collect CSV Ingest instances data"
+    order = pyblish.api.CollectorOrder + 0.1
+    hosts = ["traypublisher"]
+    families = ["csv"]
```

Comment:
Opening discussion: Can we define the family more specificaly, like `csveditorial`?

Reply:
Just found out the creator actually has family `editorialcsv`, why is used only `"csv"` during publishing?

Reply:
Yeah it might be confusing but following is the difference between those two families:
`editorialcsv`: main product witch is holding csv files to be published as backup during an iteration
`csv`: additional family on instances generated from each raw found in csv files. Those could be main family `plate` but needs also additional family `csv` so this plugin is activated and collects additional instance.data _pre-collected_ in creating.

Reply:
> editorialcsv: main product witch is holding csv files to be published as backup during an iteration

Isn't the `csv` data more than just "editorial" or am I misinterpreting what "editorial" means? It might just be a bunch of assets being ingested, not just shots. Right?

Reply:
Ok, my point is to not use only `"csv"` family, rather be more specific like `"csv-place"`, as an example.